ui/app/AppLayouts/Chat/CommunityComponents/CommunityProfilePopup.qml

```diff
@@ -108,6 +108,19 @@ ModalPopup {
         popEnter: Transition { enabled: false }
         popExit: Transition { enabled: false }
 
+        Component {
+            id: inviteFriendsView
+            CommunityProfilePopupInviteFriendsView {
+                headerTitle: qsTr("Invite friends")
+                contactListSearch.chatKey.text: ""
+                contactListSearch.pubKey: ""
+                contactListSearch.pubKeys: []
+                contactListSearch.ensUsername: ""
+                contactListSearch.existingContacts.visible: profileModel.contacts.list.hasAddedContacts()
+                contactListSearch.noContactsRect.visible: !contactListSearch.existingContacts.visible
+            }
```

Comment:
^ This is not very beautiful that we have to set every single property of the child component, to make it work, but it is what it is for now.

Screams for refactor